services/blockchain-indexer/shared/dataService/business/blocks.js

```diff
@@ -218,10 +217,7 @@ const getBlocks = async params => {
 	} catch (err) {
 		// Block does not exist
 		if (err.message.includes('does not exist')) {
-			let errMessage;
-			if ('id' in params && err.message.includes(params.id)) errMessage = `Block with ID ${params.id} does not exist`;
-			if ('height' in params) errMessage = `Block at height ${params.height} does not exist (${err.message})`;
-			throw new NotFoundException(errMessage || err.message);
+			return blocks;
```

Comment:
Will execution come to this condition anymore as we `throw` statements with `return`?

Reply:
@priojeetpriyom Your question is a bit unclear. Could you please clarify?

@nagdahimanshu We can probably replace the entire catch body with the following statement:
```
if (!err.message.includes('does not exist')) throw err;
```